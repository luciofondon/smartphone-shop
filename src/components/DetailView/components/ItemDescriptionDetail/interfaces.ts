export interface IItemDescriptrionDetail {
  label: string;
  value: string;
}
