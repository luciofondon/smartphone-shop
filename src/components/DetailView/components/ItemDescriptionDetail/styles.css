.title-description {
    font-weight: bold;
    float: left;
}

.value-description {
    float: right;
}

.item-description {
    width: 600px;
    height: 30px;
}