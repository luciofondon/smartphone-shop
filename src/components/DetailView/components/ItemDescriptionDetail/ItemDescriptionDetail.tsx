import React from 'react';

import { IItemDescriptrionDetail } from './interfaces';
import './styles.css';

function ItemDescriptrionDetail({ label, value }: IItemDescriptrionDetail) {
  return (
    <div className="item-description">
      <div className="title-description">{label}</div>
      <div className="value-description">{value}</div>
    </div>
  );
}

export default ItemDescriptrionDetail;
