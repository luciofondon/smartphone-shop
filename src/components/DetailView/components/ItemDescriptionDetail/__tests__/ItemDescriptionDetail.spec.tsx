import React from 'react';
import { render } from '@testing-library/react';

import ItemDescriptionDetail from '../ItemDescriptionDetail';

test('render ItemDescriptionDetail', () => {
  const { container } = render(<ItemDescriptionDetail label="Test" value="Test value" />);
  expect(container).toMatchSnapshot();
});
