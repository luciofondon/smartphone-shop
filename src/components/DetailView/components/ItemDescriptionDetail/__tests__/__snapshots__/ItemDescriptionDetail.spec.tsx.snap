// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ItemDescriptionDetail 1`] = `
<div>
  <div
    class="item-description"
  >
    <div
      class="title-description"
    >
      Test
    </div>
    <div
      class="value-description"
    >
      Test value
    </div>
  </div>
</div>
`;
